connector/spanmetricsconnector/connector.go

```diff
@@ -520,11 +522,20 @@ func (p *connectorImp) buildAttributes(serviceName string, span ptrace.Span, res
 	if !contains(p.config.ExcludeDimensions, statusCodeKey) {
 		attr.PutStr(statusCodeKey, traceutil.StatusCodeStr(span.Status().Code()))
 	}
+
 	for _, d := range dimensions {
 		if v, ok := utilattri.GetDimensionValue(d, span.Attributes(), resourceAttrs); ok {
 			v.CopyTo(attr.PutEmpty(d.Name))
 		}
 	}
+
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScope.Name()) && instrumentationScope.Name() != "" {
```

Comment:
I more recommand to use `IncludeInstrumentationScope`.

Reply:
In most cases (when exclude_instrumentation_scope is empty), your PR would allow any scope attribute to be converted into a spanmetrics label, which is not what we expect.